directions/main.go

```diff
@@ -81,3 +78,17 @@ func (c *Commute) GetMapsURL() string {
 
 	return url
 }
+
+func AddressIsValid(address string) bool {
+	client, err := maps.NewClient(maps.WithAPIKey(MAPS_API_KEY))
+	utils.Check(err)
```

Comment:
Could the creation of the client be moved outside of the function and to the top? Then we can move it out of getInfo too. Since we only need one instance of the client for this package.